src/dotnet/commands/dotnet-publish/Program.cs

```diff
@@ -43,6 +43,11 @@ public static int Run(string[] args)
             CommandOption versionSuffixOption = app.Option(
                $"--version-suffix <{LocalizableStrings.VersionSuffixOption}>", LocalizableStrings.VersionSuffixOptionDescription,
                 CommandOptionType.SingleValue);
+
+            CommandOption profileProjOption = app.Option(
+               $"--profile <{LocalizableStrings.ProfileProjOption}>", LocalizableStrings.ProfileProjOptionDescription,
```

Comment:
I am not picky about names, as far as the implementation goes it is a filter, as far as the providers are concerned it is a profile. But as far as the end users are concerned it is a filter.  I will change it to --filter then